### ISSUE TITLE
FAWE deadlocking the server

### ISSUE THREAD

**Issue by iLemon** [IntellectualSites/FastAsyncWorldEdit#391]
# Bug report for FastAsyncWorldEdit 1.15.2

**[REQUIRED] FastAsyncWorldEdit Configuration Files:** 

https://athion.net/ISPaster/paste/view/0102cf6ec4454bb48c998d87d6234d7a

**[REQUIRED] FastAsyncWorldEdit Version Number:** 

```
> fawe ver
[19:18:01 INFO]: FastAsyncWorldEdit-157 created by Empire92
[19:18:01 INFO]: FAWE version:
 - Date Mar 31, 2020, 12:00:00 AM
 - Commit fc6e0235
 - Build 157
 - Platform Bukkit
[19:18:01 INFO]: Wiki: https://github.com/IntellectualSites/FastAsyncWorldEdit-1.13/wiki
[19:18:01 INFO]: ------------------ Platforms ------------------
* Bukkit-Official(1.15-157;391fdcb)

[19:18:01 INFO]: ---------------- Capabilities ----------------
GAME_HOOKS: Bukkit-Official
CONFIGURATION: Bukkit-Official
USER_COMMANDS: Bukkit-Official
PERMISSIONS: Bukkit-Official
WORLDEDIT_CUI: Bukkit-Official
WORLD_EDITING: Bukkit-Official
```
**[REQUIRED] Spigot/Paper Version Number:** 

```
> ver
[19:18:18 INFO]: This server is running Tuinity version git-Tuinity-4bd09e4 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)
```
**[REQUIRED] Description of the problem:**

FAWE is causing deadlocks and freezing the server frequently (every few hours this server freezes)

**How to replicate**:

Not sure why it happens. Just have a 1.15 server with the same versions and 50-100 players with some of them using worldedit?

See the blocked threads here: https://gist.githubusercontent.com/iLemon/c83dae53539512dc52359578b2419c00/raw/15051ccf5ae767de8ee312d9436b8dac3e07f90d/gistfile1.txt

**Comment by MattBDev** [comment]
@iLemon Can you please try to reproduce this using build 158 (it should be available shortly after I post this comment)?

**Comment by aurorasmiles** [comment]
Since there is no reply here for over a month now and I didn't hear of any similar problem, can this be closed?

**Comment by aurorasmiles** [comment]
Believing this was fixed with 004a60d, closing
